src/pudl/transform/ferc714.py

```diff
@@ -533,3 +541,40 @@ def out_ferc714__hourly_planning_area_demand(
         df[columns], table_name="out_ferc714__hourly_planning_area_demand"
     )
     return df
+
+
+@asset(
+    io_manager_key="parquet_io_manager",
+    op_tags={"memory-use": "high"},  # Should this be high?
+    compute_kind="pandas",
+)
```

Comment:
* I believe this asset should use the `pudl_io_manager` -- in general we send all output to both SQLite and Parquet. Only the hourly tables (which can run into the hundreds of millions of rows) are written to Parquet alone.
* I don't think this asset will be high memory -- with just annual data and 10 years of forecasts for each planning area it'll be a pretty small table.

Reply:
Making these updates now.

In terms of when to use `pudl_io_manager` vs `parquet_io_manager`, is it basically a judgment call/depends on how the dev work is going? As in, if the transformation we're trying to run is going to output data of at least a certain size, then we go with parquet? Similar question for tags like `memory-use` - it's sort of on a case-by-case basis?

Reply:
Right now we use `pudl_io_manager` for everything with the exception of the hourly time series, which can be quite long.  For hourly tables we use `parquet_io_manager` Not all of the hourly tables really need to only be in Parquet, but enough of them do that it seemed simpler to make this clear distinction.

"High" memory use is any transform that has a peak memory usage that's higher than one CPU's fair share of the memory on the nightly build machine (16 CPUs and 64GB of memory, so anything larger than 4GB).  Identifying which assets fall into this category is haphazard right now -- I basically ran all of the ones I thought might be high memory and watched resource consumption on my laptop.  I'm sure we could do some real profiling and get a more comprehensive answer.